pandas/tseries/offsets.py

```diff
@@ -182,6 +182,49 @@ def __add__(date):
     date + BDay(0) == BDay.rollforward(date)
 
     Since 0 is a bit weird, we suggest avoiding its use.
+
+    Parameters
+    ----------
+    n : int (default 1)
+    normalize : bool (default False)
+
+    Parameters that ADD to the offset (like Timedelta)
```

Comment:
@TomAugspurger @jorisvandenbossche 

will this work formatting wise? (prob needs even more in this doc-string, but this is a good start)

Reply:
It will not work formatting wise for sphinx / online docs, I think (should try out to be sure, but I think numpydoc does not accept custom sections)

Reply:
Alternative would be to do like (but then with all options listed where I put `...`):

```
Parameters
----------
...
years, months, weeks, ... nanoseconds : int, optional
    Parameters that ADD to the offset (like Timedelta)
year, month, week, ..., nanosecond : int, optional
    Parameters that REPLACE the offset value
```

Reply:
I think that recent numpydocs raise if there are extra sections, so this won't work.